README.md

```diff
@@ -98,9 +99,10 @@ func main() {
 		syn      = true
 		proto    = "tcp"
 		ip       = "192.168.1.120"
+                proxy = "socks5://127.0.0.1:1080"
 	)
 
-	scan, err := gomap.ScanIP(ip, proto, fastscan, syn)
+	scan, err := gomap.ScanIP(ip, proto, fastscan, syn, "")
```

Comment:
Setting the var and then just passing a empty string and leaving the var unused I pretty hacky